### DISCUSSION TITLE
How does the app check for .NET on the system and install if not found?

### DISCUSSION THREAD

**Discussion by yesseruser** [Tyrrrz/YoutubeDownloader#535] [General]
I haven't been able to find anything in Program.cs, is it inside the source code?

**Comment by Tyrrrz** [comment]
This is no longer the case, new versions of the app come bundled with the runtime, so no installation on the side of the end-user is required. This became viable after migrating from WPF to Avalonia for the GUI frontend.

Previous versions used this: https://github.com/Tyrrrz/DotnetRuntimeBootstrapper
